web/webpack.config.js

```diff
@@ -76,7 +82,7 @@ const copyWebpackCommands = [
     from: `${STATIC_ROOT}/../custom/homepages/v2/announcement`,
     to: `${DIST_ROOT}/announcement`,
   },
-];
+].filter((f) => fs.existsSync(f.from));
```

Comment:
Added a filter here for any files that only exist internally.  I was going to add a note here, but I'm not sure of the full context.  Let me know if there's a specific thing you'd like noted here.

Reply:
This should be fine.
Exploring alternatives, maybe https://webpack.js.org/plugins/copy-webpack-plugin/#filter would be cleaner, which if works (documentation didn't say what it is filtering), we can also remove the 2 other `fs.existsSync` calls outside of that `copyWebpackCommands[]`

Reply:
Yeah, that sounds right to me. Would you like to log that in another ticket or have that done as part of this PR?

Reply:
No worries -- this is good enough :+1: